Newsletters/Yearn Finance Newsletter #48/Yearn Finance Newsletter #48.md

```diff
@@ -1,135 +1,133 @@
 
 ![](image1.png)
 
-Welcome to the 48th edition of the Yearn Finance Newsletter. Our aim with this newsletter is to keep the Yearn and the wider crypto community informed of the latest news, including product launches, governance changes, and ecosystem updates. If you’re interested in learning more about Yearn Finance, follow our official [Twitter](https://twitter.com/iearnfinance) and [Medium](https://medium.com/iearn) accounts.
+Bienvenidos a la edición número 48 de la newsletter de Yearn Finance. Con esta newsletter nuestra idea es mantener informada a la comunidad de Yearn y de cripto en general con las últimas noticias, incluyendo lanzamiento de productos, cambios de gobernanza y actualizaciones del ecosistema. Si estás interesado en aprender más sobre Yearn Finance, sigue las cuentas oficiales de [Twitter](https://twitter.com/iearnfinance) y [Medium](https://medium.com/iearn). 
 
-## Summary
+## **Resumen**
 
--   Yearn Goes Multichain with Fantom
-    
--   Yearn Reaches $5B TVL (Again)
+-   Yearn se convierte en multi blockchain con la ayuda de Fantom
+
+-   Yearn alcanza 5 Billones de dólares en TVL (De nuevo)
     
--   How You Can Contribute to Yearn
+-   ¿Cómo puedes colaborar con Yearn?
     
--   Call for Ideas: YFI Tokenomics Revamp
+-   Convocatoria de ideas: Nuevas Tokenomics de YFI
     
--   Transparency at Yearn
+-   Transparencia en Yearn
     
--   Vaults at Yearn
+-   Las vaults de Yearn
     
--   Ecosystem News
+-   Noticias del ecosistema
 
-## Yearn Goes Multichain with Fantom
+## Yearn se convierte en multi blockchain con la ayuda de Fantom
 
 ![](image2.png)
 
-The Realm of Multichain beckons. And Yearn has heeded the call. 
-
-Today, we go multichain with the launch of Iron Bank Fantom and the first Fantom vaults on
+El reino de multi blockchain ha hecho el llamado. Y Yearn ha acudido.
 
-https://beta.yearn.finance
+Hoy, nos convertimos en multi blockchain con el lanzamiento del Iron Bank Fantom y las primeras vaults en Fantom en https://beta.yearn.finance
 
-You may ask, “Why multichain now?”. This is because Yearn requires specific infrastructure to safely and efficiently function, both external (Gnosis safe, DEX liquidity, block explorer) and internal (frontend and backend).
+Te preguntarás, ¿por qué multi blockchain ahora?. Es porque Yearn requiere una infraestructura específica para funcionar de forma eficiente, tanto de forma externa (Gnosis safe, liquidez en DEX, explorador de bloques) e internamente (frontend y backend).
 
-Our v2 vault codebase has hardened over the past few months, and the new beta website is a vast improvement, allowing us to switch chains relatively easily, something we simply could not have done in the past.
+La base de código de nuestra vault V2 se ha fortalecido en los pasados meses, y la nueva web versión beta ha sido una mejora importante, permitiéndonos cambiar de blockchain con relativa facilidad, algo que implemente no podíamos hacer en el pasado.
```

Comment:
AS-IS: La base de código de nuestra vault V2 se ha fortalecido en los pasados meses, y la nueva web versión beta ha sido una mejora importante, permitiéndonos cambiar de blockchain con relativa facilidad, algo que implemente no podíamos hacer en el pasado.

TO-BE: La base de código de nuestra vault V2 se ha fortalecido en los pasados meses, y la nueva web versión beta ha sido una mejora importante, permitiéndonos cambiar de blockchain con relativa facilidad, algo que simplemente no podíamos hacer en el pasado.